ffi/wasm/lib/high-level.mjs

```diff
@@ -40,24 +40,99 @@ export class FixedDecimal {
     icu4x.icu4x_fixed_decimal_negate(this.underlying);
   }
 
-  write_to(writable) {
-    icu4x.icu4x_fixed_decimal_write_to(this.underlying, writable.underlying);
+  toString() {
+    const writable = icu4x.icu4x_buffer_writeable_create(0);
+    try {
+      icu4x.icu4x_fixed_decimal_write_to(this.underlying, writable);
+
+      const outStringPtr = icu4x.icu4x_buffer_writeable_get_bytes(writable);
+      const outStringLen = icu4x.icu4x_buffer_writeable_len(writable);
+      return readString(outStringPtr, outStringLen);
+    } finally {
+      icu4x.icu4x_buffer_writeable_destroy(writable);
+    }
+  }
+}
+
+const LocaleRegistry = new FinalizationRegistry(ptr => {
```

Comment:
Question: Is it conventional style to name globals like this with PascalCase?

Reply:
I thought of the registry as effectively a singleton, and since `Locale` is referring to a type I decided to keep it capitalized.

Reply:
Since we're introducing JS code for the first time into this project, we should stick to a style guide.  I figure we should use the Google JS style guide.

https://google.github.io/styleguide/jsguide.html#naming